### ISSUE TITLE
availableIncludes must be an array

### ISSUE THREAD

**Issue by afetting** [pterodactyl/panel#4027]
### Current Behavior

![image](https://user-images.githubusercontent.com/28079816/160449292-be5616a1-3abf-44e1-8add-14f2134a295b.png)
API call seems to fail with a 500 error. 

### Expected Behavior

No error should be observed

### Steps to Reproduce

Happened when updating from panel 1.6 to 1.7.
On fresh install error still persists.

### Panel Version

1.7

### Wings Version

1.6

### Games and/or Eggs Affected

All

### Error Logs

```bash
https://ptero.co/jokezutymy
http://bin.ptdl.co/0p5ct
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by mason-rogers** [comment]
Have you made any modifications to your panel?

Can't reproduce this and the code is correct

https://github.com/pterodactyl/panel/blob/develop/app/Transformers/Api/Client/ServerTransformer.php#L24

**Comment by afetting** [comment]
No modifications. It is on a fresh install of Ubuntu. I guess I will try again with another fresh install.

**Comment by DaneEveritt** [comment]
You most likely ran a dependency upgrade, not install. This is not a bug, there is a user error somewhere in the process.

**Comment by SignalWhisperer** [comment]
I just did a fresh install and I'm getting the same error.
PHP 8.1.4

**Comment by SignalWhisperer** [comment]
PHP message: PHP Fatal error:  Type of Pterodactyl\Transformers\Api\Client\ServerTransformer::$availableIncludes must be array (as in class League\Fractal\TransformerAbstract) in /var/www/pterodactyl/app/Transformers/Api/Client/ServerTransformer.php on line 14PHP message: PHP Fatal error:  Type of Pterodactyl\Transformers\Api\Client\ServerTransformer::$availableIncludes must be array (as in class League\Fractal\TransformerAbstract) in /var/www/pterodactyl/app/Transformers/Api/Client/ServerTransformer.php on line 14

**Comment by parkervcp** [comment]
@SignalWhisperer php 8.1 isn't supported. We call out that 7.4 or 8.0 are what is required in the documentation.

**Comment by SignalWhisperer** [comment]
Got it. Maybe rewording the documentation to say it must be 8.0 and not 8.1. The way it's written makes it seem like it was written before 8.1 came out and it was not updated. It doesn't explicitly mention 8.0 is required, but rather recommended.

**Comment by parkervcp** [comment]
The list is very specific. 8.1 isn't even mentioned.

![image](https://user-images.githubusercontent.com/1207679/163300507-57bb5717-478b-4b2a-96c6-d8f1166a36d1.png)

**Comment by SignalWhisperer** [comment]
"Recommended" does not mean it must exclusively be that. It leads to confusion as it is a recommendation and not a requirement. It's also listed under "Dependencies" and not "Requirements".

Could be clearer by saying "Only PHP `7.4` or `8.0` (recommended) are supported".

**Comment by CHn83** [comment]
Hello, after several attempts to install Pterodactyl on a clean machine, I have the same problem as explained previously. My PHP version is 8.0
